route/car-router.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const Router = require('express').Router;
+const jsonParser = require('body-parser').json();
+const createError = require('http-errors');
+const debug = require('debug')('parkify:car-router');
+
+const bearerAuth = require('../lib/bearer-auth-middleware.js');
+const Car = require('../model/car.js');
+
+const carRouter = module.exports = Router();
+
+carRouter.post('/api/car/:carID', bearerAuth, jsonParser, function(request, response, next) {
+  debug('POST: /api/car/:carID');
+
+  if (Object.keys(request.body).length === 0) return next(createError(400, 'Bad Request'));
+
+  request.body.userID = request.user._id;
+
+  new Car(request.body).save()
```

Comment:
`create()` does this [http://mongoosejs.com/docs/api.html#model_Model.create](http://mongoosejs.com/docs/api.html#model_Model.create)